spec/microservices_engine/initializer_spec.rb

```diff
@@ -26,16 +28,41 @@
     end
     context 'otherwise' do
       it 'checks for a config file and parses it as YAML' do
+        stub_request(:post, "http://example.com/services/register")
+          .to_return(body: response_data.to_json)
         stub_const 'MicroservicesEngine::Initializer::CONFIG_FILE', 'apples'
         expect(described_class).to receive(:check_for_config_file!)
           .with 'apples'
         expect(YAML).to receive(:load_file).with('apples')
           .and_return config_data
         call
       end
-      it 'checks in with the router using the data from the config file'
-      it "raises an exception if the router's response code was not 200"
-      it 'updates the connections database using the response from the router'
+      it 'checks in with the router using the data from the config file' do
+        allow(described_class).to receive(:check_for_config_file!) { nil }
```

Comment:
Of course, 'and_return' is totally used elsewhere in there, so I don't know why I didn't just stay consistent. 😛